src/App.jsx

```diff
@@ -1,7 +1,53 @@
-import data from "./data.json";
+import { Album } from "./components/Album"
+import data from "./data.json"
 
-console.log(data);
+const selectAlbums = data.albums.items
 
-export const App = () => {
-  return <div>Find me in src/app.jsx!</div>;
+export const App = () =>{
+return(
+  <div>
```

Comment:
Remember to indent and add spaces properly. Also consider using a React fragment instead of a div, if it's just for wrapping purposes
```
export const App = () => {
  return (
    <>
```